packages/lesswrong/components/notifications/NotificationsMenu.jsx

```diff
@@ -2,43 +2,59 @@ import { Components, registerComponent, withList } from 'meteor/vulcan:core';
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Drawer from 'material-ui/Drawer';
-import Badge from 'material-ui/Badge';
+import Badge from '@material-ui/core/Badge';
 import {Tabs, Tab} from 'material-ui/Tabs';
 import AllIcon from '@material-ui/icons/Notifications';
 import ClearIcon from '@material-ui/icons/Clear';
 import PostsIcon from '@material-ui/icons/Description';
 import CommentsIcon from '@material-ui/icons/ModeComment';
 import MessagesIcon from '@material-ui/icons/Forum';
+import { withStyles } from '@material-ui/core/styles';
 import withUser from '../common/withUser';
+import classNames from 'classnames';
 
 // import { NavDropdown, MenuItem } from 'react-bootstrap';
 import Notifications from '../../lib/collections/notifications/collection.js'
 
-
-const badgeContainerStyle = {
-  padding: 'none',
-}
-const badgeStyle = {
-  backgroundColor: 'none',
-  color: 'rgba(0,0,0,0.6)',
-  fontFamily: 'freight-sans-pro, sans-serif',
-  right: "-16px"
-}
-
 const tabLabelStyle = {
   color: "rgba(0,0,0,0.8)",
   fontFamily: "freight-sans-pro, sans-serif"
 }
 
-const iconStyle = {
-  color: "rgba(0,0,0,0.8)",
-}
-
-const cancelStyle = {
-  color: "rgba(0,0,0,0.3)",
-  margin: "10px",
-  cursor: "pointer"
-}
+const styles = theme => ({
+  badgeContainer: {
+    padding: "none",
+    fontFamily: 'freight-sans-pro, sans-serif',
+    verticalAlign: "inherit",
+  },
+  badge: {
+    backgroundColor: 'inherit',
+    color: 'rgba(0,0,0,0.6)',
+    fontFamily: 'freight-sans-pro, sans-serif',
```

Comment:
This is actually more important to fix. We no longer use freight-sans-pro, so it would be good to no longer rely on it. Might still be fine to keep as long as we add an issue for it.

Reply:
Removed all freight-sans-pro from this file. It wasn't quite a no-op (the font of the number-of-PMs badge changed to Calibri, specified in Material-UI's stylesheet) but it certainly didn't make anything worse.

There are still freight-sans-pro references in `FormSubmit`, `NotificationsItem`, `NotificationsMenuButton`, and `alignmentForumTheme.js`. I assume the last one is legit and the other three aren't.